Sources/ScrollingTabController.swift

```diff
@@ -56,6 +56,17 @@ public class ScrollingTabController: UIViewController, UIScrollViewDelegate, UIC
             tabView.centerSelectTabs = centerSelectTabs
         }
     }
+
+    /// Specifies the height of the top tab bar. Defaults to 44.0
+    public var tabBarHeight: CGFloat = 44.0 {
+        didSet {
+            guard tabBarHeightConstraint != nil else {
+                return
+            }
```

Comment:
why not bind? of course it would be terrible of client code to set this on a background thread while `viewDidLoad` is being called, but... :)